README.md

```diff
@@ -89,8 +101,36 @@ The status of an API is determined by availability.
 
 ## open-api.yml or service.wsdl
 
-A WSDL or Open API document for the API.
- > NOTE: guidance for these items is still being developed.
+Provide an **open-api.yml** file for a REST API or a **service.wsdl** file for a SOAP webservice.
+
+Standard formats such as [WSDL][WSDL] and [Open API][OpenAPI] provide consumers with enough information to start
+developing client applications, even without access to the actual API.  
+They also allow us to generate more detailed documentation in the HTML view of the catalogue.
+
+>**NOTE**  
+We recommend that you don't include the actual url/endpoint of your api servers, instead replace it with `https://private.url`  
+In the HTML view of the catalogue the URL will appear as `https://sandbox.api.nhs.wales/{system-id}/{api-id}/`
+
+#### Providing an Open API document
+For Open API documents, the description of each _path/operation_ is used to generate documentation in the HTML view of the cataloge.
```

Comment:
Again, perhaps use an active voice and fix typo [replace cataloge with catalogue]

Reply:
Not sure about active voice as a blanket rule. I'd say we should select what most clearly communicates the key information.

With that in mind, consider these options:
1. Passive: _the description [...] is used_
2. Active: _we use the description..._
3. Active: _our catalogue processor uses the description_

In Option 1, I would say the passive voice hints at the existence of an automated process (i.e. no _active_ human involvement) without dwelling on what that process is.
Option 2 hints at a more manual process and slightly distracts from the key information we want to communicate here - the descriptions and how they're used, not about who or what acts upon them.
Option 3 is explicit but further distracts from the key point


```suggestion
For Open API documents, the description of each _path/operation_ is used to generate documentation in the HTML view of the catalogue.
```